osquery/CMakeLists.txt

```diff
@@ -33,6 +33,12 @@ else()
     -Wno-unused-parameter
     -Wno-gnu-case-range
   )
+  if (CMAKE_CXX_COMPILER MATCHES "clang")
```

Comment:
Are we guaranteed to be on clang at this point? I added it since if someone were to set `CC=gcc` as an environment variable, `-flto` would cause issues.